edb/server/pgcon/pgcon.pyx

```diff
@@ -929,7 +941,18 @@ cdef class PGProto:
 
         elif mtype == b'A':
             # NotificationResponse
-            self.buffer.discard_message()
+            self.buffer.read_int32()  # discard pid
+            channel = self.buffer.read_null_str().decode()
+            payload = self.buffer.read_null_str().decode()
+            self.buffer.finish_message()
+
+            if channel == '__edgedb_ddl__':
```

Comment:
We should log if an unsupported notification arrives. EdgeDB users cannot access Postgres directly so they won't be using those themselves, no? And even if they did, we should at least log if the channel starts with `__edgedb_`.

Reply:
> We should log if an unsupported notification arrives.

They shouldn't. We explicitly listen to the `__edgedb_ddl__` channel. But I'll add logging anyways just in case and for future debug.

> EdgeDB users cannot access Postgres directly so they won't be using those themselves, no?

No, we don't expose this yet.